pkgs/by-name/co/comfyui/fetch-model.nix

```diff
@@ -0,0 +1,61 @@
+# This is a fetcher made specifically for fetching models to be used by
+# ComfyUI.
+{ lib, pkgs }:
+{ url
+, format
+, bearer ? null
+, bearerFile ? null
+,  ...
+}@args: let
+  name = (lib.replaceStrings
+    ["?" "&" ":" "/"]
+    ["__questionmark__" "__ampersand__" "__colon__" "__slash__"]
+    url
```

Comment:
Note that this custom name will affect the `outputHash`. In particular, if you download the same safetensors file from two different URLs they'll end up in two different store paths

Reply:
Good catch!  Is there a general pattern established on how to get files named from `fetchurl`?  I've only found a function that was removed from `nixpkgs`, but I can't remember what it was at the moment.